azure-pipelines.yml

```diff
@@ -0,0 +1,27 @@
+pool:
+  name: Hosted VS2017
+
+variables:
+  SignType: 'Test'
```

Comment:
Actually I like that this is test signing, because I've broken signing before with a PR.